src/Angor/Client/Pages/Founder.razor

```diff
@@ -34,14 +34,19 @@
 
                 </div>
             </div>
-            <button class="btn btn-border" @onclick="NavigateToCreateProject" disabled="@scanningForProjects">
-                <i>
-                    <Icon IconName="add"></Icon>
-                </i>
-                <span class="nav-link-text ms-1">
-                    @(scanningForProjects ? "Scanning..." : "Create Project")
-                </span>
-            </button>
+            <div class="tooltip-container" data-bs-toggle="tooltip" title="@GetCreateButtonTooltip()">
```

Comment:
can you make a separate pr just for issue 166?

Reply:
done, 
but there's also a check that is a corelated to 166 in createtransaction that checks how many keys are already existing, and if there are more than 15 then fail.

(because of an issue like listed above)

Reply:
i'll also add that its needed because disabling the button in founder.razor isn't good enough, 
because people can "hack" and change the URL.
so there needs to be check in the createtransaction (or a check on oninitiallized in create.razor that doesn't show the create module if you have more than 15.)